docs/pages/community.md

```diff
@@ -10,13 +10,11 @@ We recommend you choose one, and not more than one.
-
-| Name                                                  | Short description                                                                        | 🚦  | Used by                                                                        |
-| ----------------------------------------------------- | ---------------------------------------------------------------------------------------- | :-: | ------------------------------------------------------------------------------ |
-| [RPH slack](ucl-programming-hub.slack.com)            | UCL's research programming community on Slack Instant.                                   | 🟢  |                                                                                |
-| [Slack](https://slack.com/intl/en-gb/)                | Instant messaging platform.                                                              | 🟠  |                                                                                |
-| [Discussions](https://docs.github.com/en/discussions) | Discussion forum built into GitHub repositories.                                         | 🟠  | [SciKit-Surgery](https://github.com/SciKit-Surgery/scikit-surgery/discussions) |
-| [Discourse](https://discourse.org/)                   | Discussion forum.                                                                        | 🟠  | [napari](https://forum.image.sc/tag/napari)                                    |
-| [Gitter](https://gitter.im/)                          | Open-source instant messaging platform. Links directly to GitLab or GitHub repositories. | 🟠  | [Matplotlib](https://gitter.im/matplotlib/matplotlib)                          |
+| Name                                                         | Short description                                                                                                                                                                                                                                                                                     | 🚦  | Used by                                                                        |
+| ------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-: | ------------------------------------------------------------------------------ |
+| [Slack](https://slack.com/intl/en-gb/)                       | Instant messaging platform. For projects with UCL, any University of London, and UCL-ARC collaborators we have the [Research Programming Hub](ucl-programming-hub.slack.com) Slack channel. Any member of ARC can create a project channel. Note, however, the limitation for external collaborators. | 🟢  |                                                                                |
```

Comment:
What's the limitation for external collaborators? What recommendation we have if an external collaborator wants to join UCL-ARC?